REPORT_BUG.md

```diff
@@ -0,0 +1,36 @@
+# How to report a bug
+
+*Tickets -be it bug reports or feature requests- are the basics of any software improvement. The lack of tickets or poorly written ones seriously impair a healthy software development. Like building a house on weak foundations, this should be avoided.*
```

Comment:
I would change `-be` to `-- be` for clarity that a long dash is intended. Likewise for `requests-`, should be `requests --`

Reply:
Ah, also it should be "be they bug reports or feature requests", since everything is plural.

Reply:
I would say "a healthy software development process" (or "environment").